### ISSUE TITLE
clarify exactly what should be ingested from NEA "planets" vs "composite planets" 

### ISSUE THREAD

**Issue by zkbt** [zkbt/exoatlas#36]

**Comment by zkbt** [comment]
Right now (in version 0.3.3), the default set of parameters is being ingested from the "planets" table from the NASA Exoplanet Archive. This is good in that parameters are self-consistent within planets. This is bad because some recent papers don't publish all parameters, so important information might be missing for a particular planet. (This appears to be especially true for some nearby planets that folks care particularly about and therefore publish more papers on, making small updates to their properties).

**Comment by zkbt** [comment]
It *might* be that we just want to take the "composite" planets table instead but mask out "calculated value" (= not actually measured with observations) masses and radii. But, we should think carefully about what exactly we want to be using this for and therefore what decisions we should be making about what gets imported.

**Comment by zkbt** [comment]
I've switched over to Planetary Systems Composite Parameters but also implemented options to merge in values from individual references with `.update_reference` or to update individual values with `update_values`. I still need to do a little bit more exploration to find what messes might be caused by using the PSCP table, but I think it might be best simply to let those emerge as we play with it, instead of trying to do everything perfectly proactively from the start ;-).

Implemented in #50 as of version 0.4.0.
